app/controllers/site_config_controller.rb

```diff
@@ -6,24 +6,20 @@ def index
   end
 
   def update
-    SiteConfig.document.update_attributes(
-      notice_fingerprinter: filtered_update_params)
+    SiteConfig.document.
+      update(notice_fingerprinter: filtered_update_params.to_h)
+
     flash[:success] = 'Updated site config'
+
     redirect_to action: :index
   end
 
 private
 
   def filtered_update_params
-    params.
-      require(:site_config).
+    params.require(:site_config).
```

Comment:
Why is this syntax changing?

Reply:
I debug it. Trying to understand why it not work and reformat. I will rollback.